### PR TITLE
Add protos for summary in Trends page.

### PR DESCRIPTION
**Pull request by luluz66**

### CODE REVIEW

File: proto/stats.proto [hunk 1]
```diff
@@ -121,6 +121,22 @@ message TrendStat {
   int64 branch_count = 20;
 }
 
+message Summary {
+  // The total number of builds from a specific period.
+  int64 num_builds = 1;
+
```
Comment (jdhollen) [inline proto/stats.proto:127]: what happened to field #2 :)

Comment (luluz66) [reply proto/stats.proto:127]: Done.

File: proto/stats.proto [hunk 2]
```diff
@@ -143,6 +159,11 @@ message GetTrendResponse {
   repeated ExecutionStat execution_stat = 3;
 
   bool has_invocation_stat_percentiles = 4;
+
+  // A summary of metrics from the period specified in the query.
+  Summary summary_current = 5;
+  // A summary of metrics from the previous period.
```
Comment (jdhollen) [inline proto/stats.proto:165]: maybe just add a little bit here about this period being equal in length to the one the user requested.

**Review by jdhollen — APPROVED** [review]

File: proto/stats.proto (hunk 2 above)
Comment (luluz66) [reply proto/stats.proto]: Done.

**Pull request merged**